scripts/cargo.sh

```diff
@@ -1,14 +1,27 @@
 #!/bin/sh
 
+# $1 -> @CURRENT_SOURCE_DIR@ as replaced by meson, we cd into it
```

Comment:
This isn't current source dir for the xi-syntect-plugin

Reply:
can't process this one

Reply:
In meson.build we have two buils targets which invoke this: `cargo-build-gxi` and `cargo-build-syntect`. In the former your comment holds true, but in the latter case we use `syntect_build_path`